app/views/layouts/_generic.njk

```diff
@@ -1,17 +1,40 @@
 {% extends "template.njk" %}
 
 {% block head %}
-  <!--[if !IE 8]><!-->
-    <link rel="stylesheet" href="/public/app.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/themes/prism.min.css" integrity="sha256-vtR0hSWRc3Tb26iuN2oZHt3KRUomwTufNIf5/4oeCyg=" crossorigin="anonymous" />
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/prism.min.js" integrity="sha256-Zb9yKJ/cfs+jG/zIOFL0QEuXr2CDKF7FR5YBJY3No+c=" crossorigin="anonymous"></script>
-  <!--<![endif]-->
-  <!--[if IE 8]>
-    <link rel="stylesheet" href="/public/app-ie8.css">
-  <![endif]-->
-  <!--[if lt IE 9]>
-    <script src="/vendor/html5-shiv/html5shiv.js"></script>
-  <![endif]-->
+
+  {% if legacy %}
+    <!--[if !IE 8]><!-->
+      <link rel="stylesheet" href="/vendor/govuk_template/govuk-template.css">
+    <!--<![endif]-->
+    <!--[if IE 8]>
+      <link rel="stylesheet" href="/vendor/govuk_template/govuk-template-ie8.css">
+    <![endif]-->
+    <link rel="stylesheet" media="print" href="/vendor/govuk_template/govuk-template-print.css">
+    <link rel="stylesheet" media="all" href="/vendor/govuk_template/fonts.css"/>
+    <!--[if !IE 8]><!-->
+      <link rel="stylesheet" href="/public/app-legacy.css">
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/themes/prism.min.css" integrity="sha256-vtR0hSWRc3Tb26iuN2oZHt3KRUomwTufNIf5/4oeCyg=" crossorigin="anonymous" />
```

Comment:
There's a fair bit of duplication between these two flows (Prism, HTML5shiv) – can we split that stuff out so that it's only in one place?